projects/plugins/super-cache/tests/e2e/package.json

```diff
@@ -23,17 +23,17 @@
 		"test:run": "jest --runInBand"
 	},
 	"devDependencies": {
-		"@jest/globals": "29.1.1",
+		"@jest/globals": "^28.1.0",
 		"@typescript-eslint/eslint-plugin": "5.36.1",
 		"@typescript-eslint/parser": "5.36.1",
```

Comment:
Are these deps needed? I don't see them being used anywhere in here.

Reply:
I was about to say "yes, these are absolutely needed" because the testing suite is set to use a typescript preset which I _thought_ imported these. But... I just reset my env and tested without these, and it's fine. So.... Removed in a9e0d7949b :)